test/e2e/snaps/test-snap-txinsights.spec.js

```diff
@@ -37,22 +36,18 @@ describe('Test Snap TxInsights', function () {
         await driver.clickElement('#connecttransaction-insights');
 
         // switch to metamask extension and click connect
-        await switchToNotificationWindow(driver, 2);
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
         await driver.clickElement({
           text: 'Connect',
           tag: 'button',
         });
 
-        await driver.waitForSelector({ text: 'Confirm' });
-
         await driver.clickElement({
           text: 'Confirm',
           tag: 'button',
         });
 
-        await driver.waitForSelector({ text: 'OK' });
```

Comment:
I believe these were added to try and combat flakiness,  I don't think we should be removing them.

cc @bowensanders

Reply:
This is replaced with `await driver.clickElementAndWaitForWindowToClose`. We can keep it but here we have have added an extra condition that the windown close when this button is clicked. In other instances of file, we wanted to make sure that the window is closed once we click the button.